### ISSUE TITLE
Add detector elements for Hodoscope pixel fibers

### ISSUE THREAD

**Issue by JeremyMcCormick** [JeffersonLab/hps-java#461]
The fiber optic cables should have detector element objects so that the ID matching works in a straightforward way and the positions can be accessed through the geometry interface rather than a hard-coded algorithm.

This is a sub-issue of #420, but the changes will be put on a separate branch so as not to disrupt development on the primary branch.

**Comment by JeremyMcCormick** [comment]
@mccaky and I agreed that this will be postponed for until after #420 is merged, as a possible enhancement.  I will leave it open for now, though, as a reminder.

**Comment by JeremyMcCormick** [comment]
Won't fix
